terraform/stacks/umccr_data_portal/workflow/main.tf

```diff
@@ -307,30 +303,26 @@ locals {
       "qc_reference_samples":[
           {
               "class": "Directory",
-              "location": "gds://production/validation_data/wts/SBJ00061_MDX190049_L1900440/analysis/dragen_wts/2021-12-01--3.9.3/SBJ00061"
-          },
-          {
-              "class": "Directory",
-              "location": "gds://production/validation_data/wts/SBJ00188_MDX190169_L1900877/analysis/dragen_wts/2021-12-01--3.9.3/SBJ00188"
+              "location": "gds://production/analysis_data/SBJ01563/wts_tumor_only/202207254fcc677a/L2200121_dragen"
           },
           {
               "class": "Directory",
-              "location": "gds://production/validation_data/wts/SBJ00199_MDX190193_L1900920/analysis/dragen_wts/2021-12-01--3.9.3/SBJ00199"
+              "location": "gds://production/analysis_data/SBJ01147/wts_tumor_only/2022071381c4257a/L2101521_dragen"
           },
           {
               "class": "Directory",
-              "location": "gds://production/validation_data/wts/SBJ00236_MDX190219_L1901023/analysis/dragen_wts/2021-12-01--3.9.3/SBJ00236"
+              "location": "gds://production//analysis_data/SBJ01620/wts_tumor_only/2022042945b6e532/L2200188_dragen"
```

Comment:
Typo: extra `/` in the path.

Also, side note -- do you recommend we clean up the "Mar 13 2022" early run transcriptome result for this https://data.umccr.org/subjects/SBJ01620?

Reply:
Just realise that Run 84 has repeat due to 10X samples reconversion. Hence marked for cleanup on early run. So no worry about it.

Reply:
Fixed typo in https://github.com/umccr/infrastructure/commit/838a0d868b45663ac7926fadbc596a083754815f